docs/version.md

```diff
@@ -67,13 +63,25 @@ In increasing order of stability, from least to most stable, these are:
     - During its availability, it must not be revised with incompatible and breaking changes.
     - It may be revised with backwards-compatible, additive changes.
 
-### Increasing stability of a release
+### Increasing stability of a resource over time
```

Comment:
Another way for us to frame this: promotion of a route requires a new route + the old one to continue to be routed.

Do we have an issue to track implementing a rule?

Reply:
Updated the framing, added #53. PTAL